java/server/test/org/openqa/selenium/grid/router/ProxyCdpTest.java

```diff
@@ -126,9 +132,40 @@ public void onText(CharSequence data) {
     socket.close();
   }
 
-  private Server<?> createBackendServer(CountDownLatch latch, AtomicReference<String> incomingRef, String response) {
+  @Test
+  public void shouldBeAbleToSendMessagesOverSecureWebSocket()
```

Comment:
Is the test supposed to be failing? I executed it locally and it did not pass. Maybe I am missing some conversation that happened in Slack?

Reply:
Yes, it is not passing, the aim of the issue mentioned in this PR is to make it work. I am working on it @diemol 

Yes, Simon and I had some discussion around it. I will be back Monday after a short staycation 😄 

I will update you on the slack thread then.

Thanks